js/main.js

```diff
@@ -1,21 +1,121 @@
+  'Шикарный дворец с вензелями и гербами в Геленджике. Кажется о нем вы когда-то видели репортаж в интернете.',
+  'Отель расположен на центральной улице города, в 1.2 км от Национального музея. Гостям предоставляются услуги консьержа и билетной кассы, а также бесплатный мини-бар (обновление мини-бара каждый день в 05: 30)',
+  'Ночлег для самых крепких духом путешественников. Тусклое помещение со странной геометрией стен, в котором более ста лет назад проживала знаменитая Кеция Мейсон',
+  'Отель Гилман Хауз будет раз принять Вас в любое время суток. Только рыбные блюда в ресторане, свежайшая рыба с Рифа Дьявола. Мы гордимся, что за всю историю не получили ни одного отрицательного отзыва.'
+];
+const PHOTOS = [
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/duonguyen-8LrGtIxxa4w.jpg',
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/brandon-hoogenboom-SNxQGWxZQi0.jpg',
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/claire-rendall-b6kAwr1i0Iw.jpg'
+];
+
+const USERS_SET = 10;
+
+const getRandomPositiveInteger = (a = 0, b = USERS_SET) => {
+  const lower = Math.ceil(Math.min(Math.abs(a), Math.abs(b)));
+  const upper = Math.floor(Math.max(Math.abs(a), Math.abs(b)));
+  const result = Math.random() * (upper - lower + 1) + lower;
+  return Math.floor(result);
+};
+
+const getRandomPositiveFloat = (a = 0, b = USERS_SET, digits = 1) => {
+  const lower = Math.min(Math.abs(a), Math.abs(b));
+  const upper = Math.max(Math.abs(a), Math.abs(b));
+  const result = Math.random() * (upper - lower) + lower;
+  return +result.toFixed(digits);
+};
+
+const getRandomMixedArray = (a, b, maxLength) => {
+  const intArray = [];
+  while (intArray.length < maxLength) {
+    const int = getRandomPositiveInteger(a, b);
+    intArray.includes(int) == 0 ? intArray.push(int) : 'continue';
   }
-  return null;
+  return intArray;
 };
 
-getRandomInt(1, 10);
+const getRandomArrayElement = (elements) => elements[getRandomPositiveInteger(0, elements.length - 1)];
+
+const getUniqueElementsArray = (array) => {
+  const uniqueNumber = getRandomPositiveInteger(a = 1, array.length-1);
+  const arrayOfIndex = getRandomMixedArray(0, uniqueNumber, uniqueNumber);
+  const newArray = [];
+  arrayOfIndex.forEach((element) => {
+    newArray.push(array[element]);
+  });
+  return newArray;
+};
 
-//gen random coordinates
-const getRandomCoordinates = (min, max, length) => {
-  if (min > 0 && max > 0 && min < max) {
-    const coordinates = (min + Math.random() * (max - min)).toFixed(length);
-    return coordinates;
+const randomArray = getRandomMixedArray(1, USERS_SET, USERS_SET);
+const getImgNumber = () => {
+  const number = randomArray.pop();
+  if (number < 10) {
+    return `0${number}`;
   }
-  return null;
+  return `${number}`;
+};
+
+const createObject = () => {
```

Comment:
```suggestion
const createOffer = () => {
```
давай так, чтобы было более специфичней